IMC_WebDev/MiniCityBlog.md

```diff
@@ -0,0 +1,96 @@
+
+To style our webpage, we will need to learn the CSS syntax. A CSS rule-set consists of a selector and a declaration block.
+<img src=https://lh3.googleusercontent.com/3UcaVdjNU8MuMn4Feims4asq1epCgIOEwJX2goOH8SZE2eP2IKz5vRvCuSL9howvCc_Co_G2TE8Tqe5pIPgTAITV9KB28hqPEWXLzvZZ7xTnF1GNBohcY75WK00KBHIfbJ4nufVR>
+
+Apart from color and font-size, these are some of the properties you will require to get started:
+
+Position: It specifies the type of positioning method used for an element: static, relative, fixed, absolute or sticky. 
+Display: it handles whether an element is shown or not. If you choose display: none, the element will not be visible.
+Background-color: It determines the background color of the element. A color can be specified by a valid color name - like “black”, a hexadecimal value - like “ffffff” or an rgb value - like “rgb(255,0,0)”.
+Height and Width: These are properties used to set the height and width of the element.
+Transform: This property is used to make 2D transformations. The most commonly used transformations are: rotate() and skew.
+rotate(): It rotates an element clockwise or counter-clockwise according to a given degree. For example, 
+transform: rotate(20deg);
+ skew(): skews an element along the X and Y-axis by the given angles. For example, this example skews the element 30 degrees along the X-axis, and 50 degrees along the Y-axis.
+transform: skew(30deg, 50deg);
+Top: It affects the vertical position of the element. The above example sets the element 18px below the top edge of its nearest positioned ancestor.
+Left:  It affects the horizontal position of the element. The above example sets the element 90px from the left edge of its nearest positioned ancestor.
+Z-Index: It specifies the stack order of an element such that an element with a greater value is drawn before an element with a lower value. For example, an element with a z-index of -990 will be drawn before an element which has a z-index of -1000.
+
+To begin with, you will need a foundation for your city to be built upon. You can use the following code as a starting off point:
+<img src=https://lh6.googleusercontent.com/nZcAdMZnQDGDfw-g03-90yQgQcdS4mhQPlOFNpuUhFpB3AIVSEF-YtZJ8n5qs332bXjnNEkxspiWQpyn1BFyTgKsEJHK-dQojxqiWR04nd_tZbHe4e-D9XLHXAjMTYuIXNFWbs50>
+
+Now that you have your foundation, you might want to start building. To help you get started, this is what the code to draw a road looks like:
+<img src=https://lh6.googleusercontent.com/sDm5NTjvOdTJLM17ABgWDYo2vwaXLoNWE1t_8fqJxEBRCHWIuUGPhvsF-SjffJrARWNaDNNJ00awF6usQsISpVBpmnEZtGlhHkIztWAmclhshrbrMZs3m0giMTLE34Cx5K2P87UA>
+
+
+Now that you’ve laid down the foundation and built a road, you can start building more roads, bridges, buildings, or anything else you might want in your mini city! This is what our city looked like: <img src=https://lh4.googleusercontent.com/yu268ztRROLzh3AjLVUeNroSMUlbOsHpZzsWNIokeRtjF7iz-1mqWCqJPdtL1-JyUX6shb3kq28_CYm1xihsf70nPM8EBfHVERg13x7ef5nf5TibItYGI7ixBu_CHcgyIUfRML_c>
```

Comment:
You should make it clear if this picture is actually the result of the code you provided or is it's the eventual goal of the blog series.